File: front/client/src/app/Models/Course/Lesson/Video/VideoResponse.ts
import { AttachmentResponse } from "src/app/Models/Attachment/AttachmentResponse";
import { Language } from "src/app/Models/enums/Language";
import { SubtitlesResponse } from "../Subtitle/SubtitlesResponse";

export interface VideoResponse {
    attachment: AttachmentResponse;
    subtitles: SubtitlesResponse[];
}

File: src/Learnify/Learnify.Core/Dto/Course/Video/VideoResponse.cs
using Learnify.Core.Dto.Attachment;
using Learnify.Core.Dto.Subtitles;
using Learnify.Core.Enums;

namespace Learnify.Core.Dto.Course.Video;

public class VideoResponse
{
    public AttachmentResponse Attachment { get; set; }
    public IEnumerable<SubtitlesResponse> Subtitles { get; set; }
}